cmd/manager/main.go

```diff
@@ -85,7 +88,18 @@ func initStaticDeps(mgr manager.Manager) {
 		glog.Fatalf("Could not create config watch: %v", err)
 	}
 
+	bootstrapKubeconfig, err := ioutil.ReadFile(*bootstrapKubeconfig)
+	if err != nil {
+		glog.Warningf("Couldn't open bootstrap kubeconfig file %v", err)
```

Comment:
Nit: generally we start error messages with a lower case letter (`couldn't open..`) and we end with `: %v`.  (...`kubeconfig file: %v`).  It gives something that looks a bit like a nested exception.